server/src/core/odoo.rs

```diff
@@ -756,7 +759,7 @@ impl SyncOdoo {
 
     pub fn add_to_rebuild_arch(&mut self, symbol: Rc<RefCell<Symbol>>) {
         if DEBUG_THREADS {
-            trace!("ADDED TO ARCH - {}", symbol.borrow().paths().first().unwrap_or(&symbol.borrow().name().to_string()));
+            trace!("ADDED TO ARCH - {} - {:?}", symbol.borrow().name(), symbol.borrow().path());
```

Comment:
@fda-odoo I had shown you a `Symbol::debug_paths()` just for this cases.
But I decided to drop it , and use this instead (leveraging the derived `debug` trait on `SymbolPath` and always displaying the symbol's name anyway). What do you think?